src/style.css

```diff
@@ -2,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 0;
+}
```

Comment:
You've correctly used the `:not(:first-child)` pseudo-class to target all but the first child. However, since you're already setting `margin-right` on `.stars__star`, you don't need to reset `margin-left` to 0 here. It's redundant and can be removed.